hack/cherry-pick.sh

```diff
@@ -126,9 +123,8 @@ function return_to_kansas {
     fi
   fi
 }
-trap return_to_kansas EXIT
+trap do_cleanup EXIT
```

Comment:
One question. `do_cleanup` relies on the `gitcleanup` variable which is set to false and after this called is set to true but this function is never called again. My question is, are we calling that method to clean something up?

Reply:
trap is like defer in go. The function is called when the script exits.

Reply:
Oh good to know I thought it worked differently :)